README.md

```diff
@@ -93,4 +94,14 @@ prisma.$use((params, next) => {
 });
 ```
 
+Sending a query without multitenancy validation require ading the `ignoreMultitenancy flag`:
```

Comment:
I think it should be "requires" with an s. Also "ading" -> "adding"

Reply:
A query = it. It requires, not require

https://forum.wordreference.com/threads/it-requires-vs-it-does-require.2926484/